app/com/linkedin/drelephant/mapreduce/data/MapReduceTaskData.java

```diff
@@ -28,13 +28,17 @@
   private long _totalTimeMs = 0;
   private long _shuffleTimeMs = 0;
   private long _sortTimeMs = 0;
+  private long _startTimeMs = 0;
+  private long _finishTimeMs = 0;
   private boolean _sampled = false;
 
   public MapReduceTaskData(MapReduceCounterData counterHolder, long[] time) {
```

Comment:
I dont like the fact that we are sending a list of times and the order decides which is what. I dont what was the reason behind this instead of just sending the times as individual parameters. I would like to change it instead of adding to the list